Game Piece Scoop

BreadBurns opened the pull request:
Finished most parts of the subsystem
Currently is set to use a CANSparkMax

BreadBurns committed:
Created a GamePieceLift file with SparkMax controller
Added REVLib json to the vendordeps
Finish working on the GamePieceLift subsystem

BreadBurns committed:
Fixed Constant names in Constants
Added some methods to the GamePieceLift subsystem

BreadBurns committed:
Fixed some variable names
Created some methods to the GamePieceLift

BreadBurns committed:
Merge branch 'main' into GamePieceLift

BreadBurns committed:
Changed the name of the subsystem to better be identified
Changed from SparkMax motors to servo motors

BreadBurns committed:
Changed some names of their constants

PNovak619 commented on src/main/java/frc/robot/subsystems/GamePieceScoop.java:
```diff
@@ -0,0 +1,37 @@
+import frc.robot.Constants;
+
+public class GamePieceScoop extends SubsystemBase {
+    /** Creates a new GamePieceLift. */
```
please change comment to refer back to subsystem name

PNovak619 commented on src/main/java/frc/robot/Constants.java:
```diff
@@ -40,5 +40,7 @@ public final class Constants {
     public static final MotorType UPRIGHT_MOTOR_TYPE = null;
     // TODO: set this to something else than 0
     public static final int LED_PWM_ID = 0;
+    public static final int GAME_PIECE_LIFT_MOTOR_2 = 0;
```
please change all game piece lift names to gamepiece scoop

BreadBurns committed:
Changed the constant names to be better fit for the subsystem

PNovak619 commented on src/main/java/frc/robot/subsystems/GamePieceScoop.java:
```diff
@@ -0,0 +1,37 @@
+public class GamePieceScoop extends SubsystemBase {
+    /** Creates a new GamePieceLift. */
+
+    private Servo gamePieceLiftMotor1;
```
please change all game piece lift names to game piece scoop

PNovak619 commented on src/main/java/frc/robot/subsystems/GamePieceScoop.java:
```diff
@@ -0,0 +1,37 @@
+        gamePieceLiftMotor2 = new Servo(Constants.GAME_PIECE_LIFT_MOTOR_2);
+        }
+
+    public void liftServo() {
```
might be better to change to servo on and servo off?

BreadBurns committed:
Changed the method names to make a little more sense

BreadBurns committed:
Changed some names in the file
Removed some unnecessary lines of code, such as the motorcontrollergroup things

PNovak619 reviewed (approved):
Nice job

BreadBurns committed:
Added a todo to see some naming info
